.github/workflows/read-all-custom-properties.yaml

```
@@ -44,7 +44,7 @@ jobs:
           ref: ${{ github.ref_name }}
 
       - name: Read All Custom Properties
-        uses: PandasWhoCode/read-all-custom-properties@1-create-an-action-to-read-all-custom-properties
+        uses: PandasWhoCode/read-all-custom-properties@2009314e217eeae8ce34792846524dfa79146e9c # v0.1.0
```

```suggestion
        uses: PandasWhoCode/read-all-custom-properties@<sha> # v1.0.0
```

We should be releasing and using a 1.0.0 version